src/ucp/rndv/proto_rndv.c

```diff
@@ -453,6 +453,7 @@ void ucp_proto_rndv_receive(ucp_worker_h worker, ucp_request_t *recv_req,
     req->send.ep                  = ep;
     req->send.rndv.remote_address = rts->address;
     req->send.rndv.remote_req_id  = rts->sreq.req_id;
+    ucp_request_send_state_clear(req, NULL);
```

Comment:
don't we set comp_cb later one of rndv_get operation?

Reply:
it failed in CI